src/Pages/CoursesPage/CoursesPage.js

```diff
@@ -20,39 +20,40 @@ export default function CoursesPage() {
         <h1>Web Development Track</h1>
       </section>
 
-      <section className="video-section basic">
-        <Link
-          to={{
-            pathname: "/video-player",
-            playlistID: "PLWKjhJtqVAbnSe1qUNMG7AbPmjIG54u88",
-          }}
-        >
-          <img src={img1} alt="Freecodecamp HTML+CSS"></img>
-        </Link>
+      <section className="video-section">  
+          <Card className="video">
+              <Link
+                to={{
+                  pathname: "/video-player",
+                  playlistID: "PLWKjhJtqVAbnSe1qUNMG7AbPmjIG54u88",
+                }}
+              >
+                <img src={img1} alt="Freecodecamp HTML+CSS"></img>
+              </Link>
+          </Card>
+ 
+          <Card className="video">
+              <Link
+                to={{
+                  pathname: "/video-player",
+                  playlistID: "PLillGF-RfqbbnEGy3ROiLWk7JMCuSyQtX",
+                }}
```

Comment:
Add somthing like
```js            
            actions={[
              <SettingOutlined key="setting" />,
              <EditOutlined key="edit" />,
              <EllipsisOutlined key="ellipsis" />,
            ]}
````


- [ ] Play button `<CaretRightOulined/>`
- [ ] Enroll Course `<PlusCircleOutlined/>`
- [ ] Open in Youtube `<YoutubeOutlined />`

[Ant Icons](https://ant.design/components/icon/)